templates/artist/artist_detail.html

```diff
@@ -104,9 +104,15 @@
             var myChart5 = new Chart(ctx, {
                 type: 'pie',
                 data: {
-                    labels: ["Me", "Other Investors", "Artist"],
+                    labels: [
+                        "Me",
+                        "Other Investors",
+                        {% for percentage_breakdown in campaign.project.artist_percentage %}
+                            {{ percentage_breakdown.name }}{% if not forloop.last %},{% endif %}
```

Comment:
Sorry, I made a mistake in my previous comment. The names here should be surrounded by quotes. ie
```javascript
{% for percentage_breakdown in campaign.project.artist_percentage %}
    "{{ percentage_breakdown.name }}"{% if not forloop.last %},{% endif %}
{% endfor %}
```